src/main/resources/portal.properties.EXAMPLE

```diff
@@ -202,3 +202,13 @@ ucsc.build=hg19
 
 # default view in oncoprint (sample, patient (default))
 oncoprint.defaultview=patient
+
+# labels to be displayed in oncoprint "Mutation color" menu for custom annotation of driver and passenger mutations in the oncoprint.
+# Set these properties to enable the respective menu options in oncoprint:
+# oncoprint.custom_driver_annotation.binary.menu_label=Custom binary annotation
+# oncoprint.custom_driver_annotation.class.menu_label=Custom tiers
+
+# set this to false to disable the automatic selection of the custom driver and passenger mutations annotation in the oncoprint 
+# (instead of OncoKB and Hotspots). By default this is true. Setting this to false will select OncoKB and Hotspots as the default 
+# annotation source instead.
+# oncoprint.custom_driver_annotation.default=false
```

Comment:
decisions:
- [ ] improve the popup by adding the annotation next to the icon. Format will be, e.g. : "Mutation V89A ( :cyclone: annotation), ( :o: annotation 2), etc". This largely solves comments 1 to 4 and 6 from @sandertan .
- [ ] for point 5 stick to tiers, but improve naming in code (make it more consistent)
- [ ] icons mentioned in point 6 will be improved and will not be simple a circle of one color. Feedback is still needed to decide on best icons.